### PR TITLE
chore(framework):Interface Design and Navigation Updates

### PR DESCRIPTION
**Pull request by ilija-**
Major
-----
- Components Landing page design update
-- Content changes and updates
-- Link routes around the buttons

Minor
-----
- Search bar design updates
- Logo added
-- Link route around the logo
-  Few icons added
- Components Menu updated

SEO
----
- Added temporary file **robots.txt** to discourage the search engines to crawl any data and content from Northlight

THEME
----
- Darkmode fixes and updates

### CODE REVIEW

File: framework/lib/components/search-bar/search-bar.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const SearchBar = forwardRef(
           defaultOptions={ filtered }
           loadOptions={ debouncedLoadOptions }
           onChange={ handleChange }
-          placeholder="Search ..."
```
Comment (SebCodesTheWeb) [inline framework/lib/components/search-bar/search-bar.tsx:100]: The changes you made to the file `search-bar.tsx` you should probably move to the `search-components-bar.tsx` which is specific to the search bar for the docs

**Review by SebCodesTheWeb — CHANGES_REQUESTED** [review]
Also, some changes that I would add would be to change height="100vh" to height="100%" on the `<Grid />` in sandbox.tsx, and put max-height="100vh" on sandbox/app/components/sub-menu on the `<Stack >` for good dark theme support

File: framework/robots.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/robots.txt:2]: It's either here or in the sandbox section. It should be in the root.

**Comment by ilija-** [comment]
> Also, some changes that I would add would be to change height="100vh" to height="100%" on the `<Grid />` in sandbox.tsx, and put max-height="100vh" on sandbox/app/components/sub-menu on the `<Stack >` for good dark theme support

This is coming with the next chore update.

File: framework/sandbox/docs/app.tsx [hunk 3]
```diff
@@ -1,159 +1,473 @@
...
-              <Capitalized sx={ { fontSize: '64px' } }>Northlight</Capitalized>
+      <HStack sx={ { bg: 'blue.500' } } borderRadius={ "md" } justifyContent="space-around">
+        <Flex
+          direction={ isNotSmallerScreen ? 'row' : 'column' }
+          gap={ 10 }
+          p={ isNotSmallerScreen ? '16' : '0' }
+        >
+
+          <Box mt={ isNotSmallerScreen ? '0' : 16 } alignSelf="flex-start" justifyContent="space-around">
+            <Fade in={ true } enterDelay={ 800 }>
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/sandbox/docs/app.tsx:68]: The eye is not that fast. You probably want to mimic "onscroll" appearance. Didn't want to lose time on coding that.

File: framework/sandbox/docs/app.tsx [hunk 4]
```diff
@@ -1,159 +1,473 @@
...
+                >
+
+                  <Box mt={ isNotSmallerScreen ? '0' : 16 } alignSelf="flex-start" justifyContent="space-around">
+                    <Fade in={ true } enterDelay={ 800 }>
+                      <H2 mb={ 6 } sx={ { color: '#111' } }>
+                        Want to join the giants ?
+                      </H2>
+                      <Box>
+                        <Capitalized>
+                          We accept only serious contributions. Be prepared for a deep screening.
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/sandbox/docs/app.tsx:449]: Haha , the content is not that revised. So far it's internal, we will change the content on the go don't worry about it.

File: framework/sandbox/docs/app.tsx [hunk 5]
```diff
@@ -1,159 +1,473 @@
...
+              </Center>
+            </Flex>
+            <H3>Design principles</H3>
+            <P>Practical guides and principles for designing and developing experiences.</P>
+            <Button rightIcon={ <ChevronRightDuo /> } variant="ghost" isDisabled={ true } bgColor="gray.50" mt={ 5 }>View here</Button>
+          </Box>
+        </SimpleGrid>
+      </Fade>
+
+      <Flex minWidth="max-content" minHeight={ 80 } alignItems="center" borderRadius={ "md" } justifyContent={ 'center' } gap="2" bgColor="gray.50">
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/sandbox/docs/app.tsx:225]: Got it.

File: framework/sandbox/docs/app.tsx [hunk 6]
```diff
@@ -1,159 +1,473 @@
...
+        <Fade in={ true } enterDelay={ 800 }>
+          <Box textAlign="center" mb={ 5 }>
+            <Capitalized sx={ { fontSize: '64px', fontWeight: 'regular', textAlign: 'center' } }>NORTH</Capitalized>
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/sandbox/docs/app.tsx:228]: True that.

File: framework/sandbox/docs/app.tsx [hunk 7]
```diff
@@ -1,159 +1,473 @@
...
+              <HStack justifyContent="space-around">
+                <Flex
+                  direction={ isNotSmallerScreen ? 'row' : 'column' }
+                  gap={ 10 }
+                  p={ isNotSmallerScreen ? '16' : '0' }
+                >
+
+                  <Box mt={ isNotSmallerScreen ? '0' : 16 } alignSelf="flex-start" justifyContent="space-around">
+                    <Fade in={ true } enterDelay={ 800 }>
+                      <H2 mb={ 6 } sx={ { color: '#111' } }>
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/sandbox/docs/app.tsx:444]: Got it.

**Comment by SebCodesTheWeb** [comment]
Screenshot 2023-02-23 at 13 02 15

This line still needs to be spread onto multiple lines

```
<Box>
<SeachComponentsBar/>
</Box>
```

**Comment by ilija-** [comment]
> 

fixed

File: framework/lib/components/search-bar/search-bar.tsx (hunk 1 above)
Comment (ilija-) [reply framework/lib/components/search-bar/search-bar.tsx:100]: Fixed.

**Review by SebCodesTheWeb — APPROVED** [review]

**Pull request merged**